nixos/modules/services/networking/wireguard.nix

```diff
@@ -240,7 +250,8 @@ in
             peers = [
               { allowedIPs = [ "192.168.20.1/32" ];
                 publicKey  = "xTIBA5rboUvnH4htodjb6e697QjLERt1NAB4mZqp8Dg=";
-                endpoint   = "demo.wireguard.io:12913"; }
+                endpoint   = "demo.wireguard.io:12913";
+                table      = "42"; }
```

Comment:
Agreed! -- I wasn't entirely sure whether omitting the new setting from the example was in the maintainers' best interests.

Rectified by 6070d91.